### PR TITLE
fix various typos

### PR DESCRIPTION
**Pull request by winstywang**

### CODE REVIEW

**Comment by piiswrong** [comment]
This still looks weird. Maybe we should change the metric interface?

**Comment by winstywang** [comment]
I am not sure whether I understand correctly. This implementation actually corresponds to the implementation of accuracy metric here:https://github.com/winstywang/mxnet/blob/4511d84ecd8ab075aa157561281579dc95fc5937/python/mxnet/metric.py#L58

File: python/mxnet/metric.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def update(self, labels, preds):
         assert len(labels) == len(preds)
         for pred, label in zip(preds, labels):
             self.sum_metric += self._feval(label, pred)
-        self.num_inst += 1
+            self.num_inst += 1
```
Comment (antinucleon) [inline python/mxnet/metric.py]: I think it should be ```self.num_inst += len(label)```

Comment (tqchen) [inline python/mxnet/metric.py]: LGTM, but why the evaluation wasn't done in batch way? i.e. _feval takes labels and preds

Comment (winstywang) [inline python/mxnet/metric.py]: I find this fix should be write. It is equivalent to += len(label) if placed outside the loop. I have tried to evaluate it in a batch way, however the tests failed. I think we should keep compatible with it. @antinucleon @tqchen @piiswrong

Comment (tqchen) [inline python/mxnet/metric.py]: OK， we can merge in as this way for now

**Pull request merged**